crates/sui-framework/deps/move-stdlib/sources/error.move

```diff
@@ -0,0 +1,81 @@
+/// This module defines a set of canonical error codes which are optional to use by applications for the
```

Comment:
Can we delete this from our stdlib?

Reply:
I would prefer not to so we can eventually let Sui devs depend on the Move stdlib from GitHub or elsewhere instead of from the Sui repo... 

I will do it for now because we already have the incompatibility via `debug`, but I do think this is worth resolving eventually.